tests/components/tibber/test_config_flow.py

```diff
@@ -0,0 +1,73 @@
+"""Tests for Tibber config flow."""
+import pytest
+
+from homeassistant.components.tibber.const import DOMAIN
+from homeassistant.const import CONF_ACCESS_TOKEN
+
+import tests.async_mock
```

Comment:
We prefer to import `AsyncMock`, `patch` etc from `tests.async_mock` to make the diff minimal in the future when we'll remove the helper. It's only needed until Python 3.8 is the minimum version.
```py
from tests.async_mock import AsyncMock, ...
```